dev/com.ibm.websphere.appserver.jsfApiStub-2.2/.project

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>com.ibm.websphere.appserver.jsfApiStub-2.2</name>
+	<comment></comment>
+	<projects>
+	</projects>
+	<buildSpec>
+		<buildCommand>
+			<name>org.eclipse.jdt.core.javabuilder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
+		<buildCommand>
+			<name>bndtools.core.bndbuilder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
+		<buildCommand>
+			<name>org.eclipse.buildship.core.gradleprojectbuilder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
+	</buildSpec>
+	<natures>
+		<nature>org.eclipse.jdt.core.javanature</nature>
+		<nature>bndtools.core.bndnature</nature>
+		<nature>org.eclipse.buildship.core.gradleprojectnature</nature>
```

Comment:
Are these gradle entries required?

Reply:
@jhanders34 didn't call this out when he first fixed the eclipse projects settings for this project (pre-revert) in #8343, so I doubt it's a problem.  Do you have an eclipse environment you can check this out in?

Reply:
I actually can't find any other projects using that `org.eclipse.buildship.core.gradleprojectnature` - I'll just delete it.